packages/ui/src/icons/firefox-icon.tsx

```diff
@@ -0,0 +1,233 @@
+import * as React from 'react';
```

Comment:
Is your IDE auto completing React as a * import? It's a default export anyway right?

Reply:
No - whenever I make a new icon, I copy/paste the first few lines from an existing icon file, which must have also had this. I'll update!

Reply:
https://github.com/facebook/react/pull/18102